README.md

```diff
@@ -111,4 +111,15 @@ dotnet ef database update LastGoodMigration
 
 Download MYSQL Workbench from [here](https://dev.mysql.com/get/Downloads/MySQLGUITools/mysql-workbench-community-8.0.17-winx64.msi)
 
-Open [cslabs-db-diagram.mwb](./cslabs-db-diagram.mwb) in the root of this project using mysql workbench.
\ No newline at end of file
+Open [cslabs-db-diagram.mwb](./cslabs-db-diagram.mwb) in the root of this project using mysql workbench.
+
+
+### Development
+
+#### Git Workflow
+
+The main branch is master, which will hold the version currently running in production.
+When some work needs to be done, you will branch off from dev using the
+naming convention `<initials>-<feature-name>` for your branch.
```

Comment:
More system are going to / instead of -.
Can you change this to `<initials>/<feature-name>`

Reply:
Updated